coderd/templateversions.go

```diff
@@ -1831,15 +1831,15 @@ func checkProvisioners(ctx context.Context, store database.Store, orgID uuid.UUI
 		return codersdk.MatchedProvisioners{}, xerrors.Errorf("provisioner daemons by organization: %w", err)
 	}
 
-	threePollsAgo := time.Now().Add(-3 * pollInterval)
+	staleInterval := time.Now().Add(provisionerdserver.StaleHeartbeats * pollInterval)
```

Comment:
At risk of sounding like a broken record, I'll echo my previous sentiment that multiplication feels wrong here. The provisioner is not really allowed to miss two heartbeats, it's only allowed to miss one and by the time we would expect the second it's already stale. This can't be a multiple of pollInterval IMO, it has to be offset from it.

Reply:
OK, so I've changed this to `provisionerdserver.StaleInterval` which is set to 90 seconds.